pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -805,24 +805,36 @@ def test_basics_nanos(self):
     @pytest.mark.parametrize(
         "value, check_kwargs",
         [
-            [946688461000000000, {}],
-            [946688461000000000 / 1000, dict(unit="us")],
-            [946688461000000000 / 1000000, dict(unit="ms")],
-            [946688461000000000 / 1000000000, dict(unit="s")],
+            [946_688_461_000_000_000, {}],
+            [946_688_461_000_000_000 / 1000, dict(unit="us")],
+            [946_688_461_000_000_000 / 1_000_000, dict(unit="ms")],
+            [946_688_461_000_000_000 / 1_000_000_000, dict(unit="s")],
             [10957, dict(unit="D", h=0)],
             [
-                (946688461000000000 + 500000) / 1000000000,
+                (946_688_461_000_000_000 + 500000) / 1_000_000_000,
                 dict(unit="s", us=499, ns=964),
             ],
-            [(946688461000000000 + 500000000) / 1000000000, dict(unit="s", us=500000)],
-            [(946688461000000000 + 500000) / 1000000, dict(unit="ms", us=500)],
-            [(946688461000000000 + 500000) / 1000, dict(unit="us", us=500)],
-            [(946688461000000000 + 500000000) / 1000000, dict(unit="ms", us=500000)],
-            [946688461000000000 / 1000.0 + 5, dict(unit="us", us=5)],
-            [946688461000000000 / 1000.0 + 5000, dict(unit="us", us=5000)],
-            [946688461000000000 / 1000000.0 + 0.5, dict(unit="ms", us=500)],
-            [946688461000000000 / 1000000.0 + 0.005, dict(unit="ms", us=5, ns=5)],
-            [946688461000000000 / 1000000000.0 + 0.5, dict(unit="s", us=500000)],
+            [
+                (946_688_461_000_000_000 + 500_000_000) / 1_000_000_000,
+                dict(unit="s", us=500000),
+            ],
+            [(946_688_461_000_000_000 + 500000) / 1_000_000, dict(unit="ms", us=500)],
+            [(946_688_461_000_000_000 + 500000) / 1000, dict(unit="us", us=500)],
+            [
+                (946_688_461_000_000_000 + 500_000_000) / 1_000_000,
+                dict(unit="ms", us=500000),
+            ],
+            [946_688_461_000_000_000 / 1000.0 + 5, dict(unit="us", us=5)],
+            [946_688_461_000_000_000 / 1000.0 + 5000, dict(unit="us", us=5000)],
+            [946_688_461_000_000_000 / 1_000_000.0 + 0.5, dict(unit="ms", us=500)],
+            [
+                946_688_461_000_000_000 / 1_000_000.0 + 0.005,
+                dict(unit="ms", us=5, ns=5),
+            ],
+            [
+                946_688_461_000_000_000 / 1_000_000_000.0 + 0.5,
+                dict(unit="s", us=500000),
+            ],
```

Comment:
IMO these don't improve legibility compared to the old ones. I'd prefer the old way to keep a single line per parameter.